packages/babel-traverse/test/replacement.js

```diff
@@ -97,4 +102,23 @@ describe("path/replacement", function() {
       );
     });
   });
+  describe("replaceWithMultiple", () => {
```

Comment:
I think this test is exactly what `babel-helper-transform-fixture-runner` does. I would suggest rewrite the unit test into fixture-style test, which is more easier to maintain. You may start from this example: `packages/babel-traverse/test/fixtures/regression/visit-tsfunctiontype-parameters`.

Reply:
Well, since this file has many other tests for the traverse API I think that it's ok 🤷